app/src/main/res/values-ar/strings.xml

```diff
@@ -266,5 +266,6 @@
     <string name="copy_right_mifos">©2016-%1$s Mifos Initiative.</string>
     <string name="all_rights_reserved">All rights reserved.</string>
     <string name="licenses">Licenses</string>
+    <string name="fail_register">Failed to Register</string>
 
```

Comment:
As we will be using `MFErrorParser.errorMessage(e)` so no need for a string. Similarly remove from **all other strings.xml** too.